kube-core/src/params.rs

```diff
@@ -473,6 +475,44 @@ impl PostParams {
 /// };
 /// let patch = Patch::Apply(&r);
 /// ```
+/// # Invalid Patches
+///
+/// In this example patch contains a `PodSpec` and **not** a complete or partial `Pod`.
+/// This patch is invalid as the full structure of a resource is required.
+/// The invalid patch will be accepted by the cluster, but no changes will be made.
+///
+/// Using serve-side style [`Apply`](Patch::Apply) patches mitigates this issue.
+///
+/// ```no_run
+/// use k8s_openapi::api::core::v1::{Pod, PodSpec};
+/// use kube::{Api, api::{PatchParams, Patch}};
+///
+/// # async fn wrapper() -> Result<(), Box<dyn std::error::Error>> {
+/// # let client = kube::Client::try_default().await?;
+/// let pods: Api<Pod> = Api::namespaced(client, "apps");
+/// let pp = PatchParams::default();
+///
+/// let invalid_patch: PodSpec = serde_json::from_value(serde_json::json!({
+///                 "activeDeadlineSeconds": 5
+/// }))?;
+///
+/// // This will have no effect on mypod.
+/// pods.patch("mypod", &pp, &Patch::Strategic(invalid_patch)).await?;
```

Comment:
As a future note to self: I am guessing this happens because `activeDeadlineSeconds` is not a field on the top level struct expected in the patch on the go side, and thus once it's deserialized, it's the apiserver is simply asked to do an empty patch, which should never be an error.

Reply:
Exactly!